README.md

```diff
@@ -20,6 +20,7 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
+- [Using Minishift with an HTTP/HTTPS Proxy](#using-minishift-with-an-proxy)
```

Comment:
I think better to have it a separate section because this is not everyone is going to use so putting it in this section will help only needy users.